tools/picker/src/index.js

```diff
@@ -7,7 +7,17 @@ import getProductsInCategory from './queries/products.graphql.js';
 
 import './styles.css';
 
-const configFile = ' https://main--aem-boilerplate-commerce--hlxsites.hlx.live/configs.json';
+/**
+ * Object containing all configuration files that should be exposed in the picker.
+ */
+const configFiles = {
+    'prod': 'https://main--aem-boilerplate-commerce--hlxsites.hlx.live/configs.json?sheet=prod',
+    'stage': 'https://main--aem-boilerplate-commerce--hlxsites.hlx.live/configs-stage.json',
+    'dev': 'https://main--aem-boilerplate-commerce--hlxsites.hlx.live/configs-dev.json',
+}
+/**
+ * Default configuration to be loaded.
+ */
 const defaultConfig = 'prod';
```

Comment:
How would a user change the config which is being used? It looks like it's hardcoded?

Do we leave it up to the project to decide, when/how to set this?

Reply:
You can select the config in the UI. This option is about the pre-selected one :)

https://github.com/user-attachments/assets/d9a4d9de-4fef-4733-9252-953660ced896

Reply:
Oh, I see, cool - so that UI already exists, since I can't see it in the changes here.

Reply:
It was there before yes. Before it was using the different sheets inside the Excel. But since we moved to different files (so they can be blocked for non-prod environments), that change was missing in the picker.